cadvisor.go

```diff
@@ -52,8 +54,41 @@ var allowDynamicHousekeeping = flag.Bool("allow_dynamic_housekeeping", true, "Wh
 
 var enableProfiling = flag.Bool("profiling", false, "Enable profiling via web interface host:port/debug/pprof/")
 
+var (
+	ignoreMetrics = metricSetValue{container.MetricSet{}}
+
+	// List of metrics that can be ignored.
+	ignoreWhitelist = container.MetricSet{
+		container.DiskUsageMetrics:       struct{}{},
+		container.NetworkUsageMetrics:    struct{}{},
+		container.NetworkTcpUsageMetrics: struct{}{},
+	}
+)
+
+type metricSetValue struct {
+	container.MetricSet
+}
+
+func (ml *metricSetValue) String() string {
+	return fmt.Sprint(*ml)
+}
+
+func (ml *metricSetValue) Set(value string) error {
+	for _, metric := range strings.Split(value, ",") {
+		if ignoreWhitelist.Has(container.MetricKind(metric)) {
+			(*ml).Add(container.MetricKind(metric))
+		} else {
+			return fmt.Errorf("unsupported metric %q specified in disable_metrics", metric)
+		}
+	}
+	return nil
+}
+
 func main() {
 	defer glog.Flush()
+	// Tcp metrics are ignored by default.
+	flag.Var(&ignoreMetrics, "disable_metrics", "comma-separated list of metrics to be disabled. Options are `disk`, `network`, `tcp`. Note: tcp is disabled by default due to high CPU usage.")
+	flag.Set("disable_metrics", "tcp")
```

Comment:
Rather than calling flag.Set, just add TCP to the default metrics on [line 58](https://github.com/google/cadvisor/pull/1159/files#diff-199ed097109d5fd1884fee60eefccfe8R58)